### PR TITLE
Update readme and contributing files

### PR DESCRIPTION
**Pull request by edisonxiang**
Why to do:
Fix some incorrect description in ReadMe file.

Prior to a PR being reviewed, there needs to be a Github issue that the PR
addresses. Replace the brackets and text below with that issue number:
For #1 

Links to the line numbers/files in the OpenStack source code that support the
code in this PR:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
-# Gophercloud: an OpenStack SDK for Go
+golangsdk: a Huawei clouds SDK for Golang
 [![Build Status](https://travis-ci.org/huawei-clouds/golangsdk.svg?branch=master)](https://travis-ci.org/huawei-clouds/golangsdk)
 [![Coverage Status](https://coveralls.io/repos/github/huawei-clouds/golangsdk/badge.svg?branch=master)](https://coveralls.io/github/huawei-clouds/golangsdk?branch=master)
 
-Gophercloud is an OpenStack Go SDK.
+golangsdk is a Huawei clouds Go SDK.
+golangsdk is based on [Gophercloud](https://github.com/gophercloud/gophercloud)
+which is an OpenStack Go SDK, and did some enhancements.
```
Comment (freesky-edward) [inline README.md]: Recommend to list all the enhancements here, so that others know what are the same as gophercloud and what are changed.

Comment (freesky-edward) [reply README.md]: I think this project used the framework of gophercloud as we think it was a great design, however the services huawei clouds support are not the same as openstack. so we will change the services supports parties.

File: README.md [hunk 2]
```diff
@@ -1,8 +1,10 @@
-# Gophercloud: an OpenStack SDK for Go
+golangsdk: a Huawei clouds SDK for Golang
 [![Build Status](https://travis-ci.org/huawei-clouds/golangsdk.svg?branch=master)](https://travis-ci.org/huawei-clouds/golangsdk)
 [![Coverage Status](https://coveralls.io/repos/github/huawei-clouds/golangsdk/badge.svg?branch=master)](https://coveralls.io/github/huawei-clouds/golangsdk?branch=master)
```
Comment (freesky-edward) [inline README.md]: I am not sure whether this could work well now.

Comment (edisonxiang) [reply README.md]: It works now :)
I will add [Go Report Card]

File: README.md (hunk 1 above)
Comment (edisonxiang) [reply README.md]: I will update it later.

**Comment by edisonxiang** [comment]
@freesky-edward  @niuzhenguo Done, PTAL.

**Comment by freesky-edward** [comment]
LGTM

**Pull request merged**